cartridges/rvw_dev_console/cartridge/templates/default/dev_console/index.isml

```diff
@@ -107,9 +107,21 @@
     <body>
         <div id="app"></div>
 
+        <!-- Setup CSRF Token -->
```

Comment:
I would prefer the token generation happens in the controller. We want to avoid isscript whenever possibke

Reply:
We can move it to an input field.  This was not an ISML form thing, so seemed like it honestly did not make a difference on how it was created, but I can make this a hidden input field.

Reply:
im more interested in the token being created in the controller. i dont care if its on the window in javascript